jdl/jhipster/client-framework-types.js

```diff
@@ -18,7 +18,7 @@
  */
 
 const Types = {
-  ANGULAR_X: 'angularX',
+  ANGULAR_X: 'angular',
```

Comment:
Constant should be removed and keep only ANGULAR

Reply:
yes